File: data/evaluation/geeks_for_geeks_successful_test_scripts/python/FIND_REPEATED_CHARACTER_PRESENT_FIRST_STRING.py
# Copyright (c) 2019-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#
def f_gold ( s ) :
    p = - 1
    for i in range ( len ( s ) ) :
        for j in range ( i + 1 , len ( s ) ) :
            if ( s [ i ] == s [ j ] ) :
                p = i
                break
        if ( p != - 1 ) :
            break
    return p


#TOFILL

if __name__ == '__main__':
    param = [
    ('ORXMflacQFBv',),
    ('39977638567848',),
    ('011110011011',),
    ('fYjfNy',),
    ('222280492',),
    ('11',),
    ('UjntBg',),
    ('6866190138212',),
    ('0000',),
    ('FWz PWEQgVlRZ',)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        if f_filled(*parameters_set) == f_gold(*parameters_set):
            n_success+=1
    print("#Results: %i, %i" % (n_success, len(param)))

File: data/evaluation/geeks_for_geeks_successful_test_scripts/java/FIND_REPEATED_CHARACTER_PRESENT_FIRST_STRING.java
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//

import java.util. *;
import java.util.stream.*;
import java.lang.*;
import javafx.util.Pair;
public class FIND_REPEATED_CHARACTER_PRESENT_FIRST_STRING{
static int f_gold ( String s ) {
  int p = - 1 , i , j ;
  for ( i = 0 ;
  i < s . length ( ) ;
  i ++ ) {
    for ( j = i + 1 ;
    j < s . length ( ) ;
    j ++ ) {
      if ( s . charAt ( i ) == s . charAt ( j ) ) {
        p = i ;
        break ;
      }
    }
    if ( p != - 1 ) break ;
  }
  return p ;
}


//TOFILL

public static void main(String args[]) {
    int n_success = 0;
    List<String> param0 = new ArrayList<>();
    param0.add("ORXMflacQFBv");
    param0.add("39977638567848");
    param0.add("011110011011");
    param0.add("fYjfNy");
    param0.add("222280492");
    param0.add("11");
    param0.add("UjntBg");
    param0.add("6866190138212");
    param0.add("0000");
    param0.add("FWz PWEQgVlRZ");
    for(int i = 0; i < param0.size(); ++i)
    {
        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
        {
            n_success+=1;
        }
    }
    System.out.println("#Results:" + n_success + ", " + param0.size());
}
}